Index: src/chapter2/exercise3.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/chapter2/exercise3.py b/src/chapter2/exercise3.py
new file mode 100644
--- /dev/null	(date 1616579354907)
+++ b/src/chapter2/exercise3.py	(date 1616579354907)
@@ -0,0 +1,9 @@
+#hours worked and time taken
+hours = 35
+time = 2.75
+
+prompt = input('Enter hours\n')
+prompt = input('Enter time\n')
+
+# Gross pay
+pay = print(hours * time)
\ No newline at end of file
